Crypto_bot_pkg/crypto_selection.py

```diff
@@ -0,0 +1,29 @@
+<<<<<<< Updated upstream
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+#This is where I'll code the functions
+
+def which_crypto():
+    print("This function will allow you to choose between BTC, ETH, ADA or DOGE coins")
+
+=======
+class crypto_bot:
```

Comment:
classes should be with capital letters (Crypto_bot), sorry for the nitpicking

Reply:
Don't be sorry, thanks for the style suggestion!